src/http_server.cc

```diff
@@ -4688,12 +4692,41 @@ HTTPAPIServer::Create(
       server, trace_manager, shm_manager, port, reuse_port, address,
       header_forward_pattern, thread_cnt, restricted_features));
 
-  const std::string addr = address + ":" + std::to_string(port);
-  LOG_INFO << "Started HTTPService at " << addr;
 
   return nullptr;
 }
 
+
+TRITONSERVER_Error*
+HTTPAPIServer::Create(
+    std::shared_ptr<TRITONSERVER_Server>& server, UnorderedMapType& data,
+    const RestrictedFeatures& restricted_features,
+    std::unique_ptr<HTTPServer>* service)
+{
+  int port;
+  int reuse_port;
+  std::string address;
+  std::string header_forward_pattern;
+  int thread_count;
+
+
+  RETURN_IF_ERR(get_value(data, "port", port));
+  RETURN_IF_ERR(get_value<int>(data, "reuse_port", reuse_port));
+  RETURN_IF_ERR(get_value(data, "address", address));
+  RETURN_IF_ERR(
+      get_value(data, "header_forward_pattern", header_forward_pattern));
+  RETURN_IF_ERR(get_value(data, "thread_count", thread_count));
+
+  TRITONSERVER_Error* err = nullptr;
+  err = Create(
+      server, nullptr, nullptr,  // TraceManager, SharedMemoryManager
+      port, (bool)reuse_port, address, header_forward_pattern, thread_count,
+      restricted_features, service);
+
+  return err;
```

Comment:
1. `return Create(...)` directly
2. Use in-line comments next to each literal like `nullptr /* trace_manager */, nullptr /* shm_manager */, ...`